### PR TITLE
Fix/ledger ibc transfer

### PR DESCRIPTION
**Pull request by alanrsoares**

### CODE REVIEW

File: app/src/components/NavSidePanel/NavSidePanel.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { defineComponent, onMounted, ref, watch, computed } from "vue";
 import { RouterLink, useRouter } from "vue-router";
 
 import { accountStore } from "@/store/modules/accounts";
-import { flagsStore } from "@/store/modules/flags";
+import { flagsStore, MARGIN_FE_URL } from "@/store/modules/flags";
```
Comment (pgoos) [inline app/src/components/NavSidePanel/NavSidePanel.tsx:5]: think it should be a part of this PR

File: app/src/components/NavSidePanel/NavSidePanelItem.tsx [hunk 2]
```diff
@@ -53,13 +54,22 @@ export default defineComponent({
...
 
+    const isMarginUrl = props.href === MARGIN_FE_URL;
+    let rel = "noopener noreferrer";
+    let target = "_blank";
+
+    if (isMarginUrl) {
+      rel = "noopener";
+      target = "_self";
+    }
+
```
Comment (pgoos) [inline app/src/components/NavSidePanel/NavSidePanelItem.tsx:65]: think it should not be a part of this PR

**Review by pgoos — CHANGES_REQUESTED** [review]
There are margin changes that should be taken out of this PR

File: app/src/components/NavSidePanel/NavSidePanelItem.tsx (hunk 2 above)
Comment (alanrsoares) [reply app/src/components/NavSidePanel/NavSidePanelItem.tsx:65]: that has been merged to develop, shouldn't be showing in this diff

Comment (alanrsoares) [reply app/src/components/NavSidePanel/NavSidePanelItem.tsx:65]: the margin changes are behind a feature flag

**Review by pgoos — APPROVED** [review]
LGTM

**Pull request merged**